test/utils/hermeticpodcertificatesigner/hermeticpodcertificatesigner.go

```diff
@@ -187,10 +189,15 @@ func (c *Controller) handlePCR(ctx context.Context, pcr *certsv1alpha1.PodCertif
 	if requestedLifetime < lifetime {
 		lifetime = requestedLifetime
 	}
-
+	host := "cluster.local"
+	if pcr.Spec.UserConfig != nil {
+		if value, exist := pcr.Spec.UserConfig[trustDomainKey]; exist {
+			host = value
+		}
+	}
```

Comment:
I know this is just test code, but I don't think we should write code that allows a user to override a spiffe trust domain, just in case someone copies this code.

Let's test the annotation by changing something less important about the certificate --- maybe change the path, or add an annotation